minutes/minutes.md

```diff
@@ -2,3 +2,4 @@
 
 * [2019-10-15](2019-10-15.md)
 * [2019-11-12](2019-11-12.md)
+* [2019-11-19](2019-11-12.md)
```

Comment:
Wrong link

Reply:
Should we reverse the order of this list so that the most recent is that top of the list?  I think that would be more intuitive, especially as we get more and more of these minutes archived.

Reply:
fixed